src/test/java/seedu/address/logic/commands/QueryPatientCommandTest.java

```diff
@@ -98,9 +97,10 @@ public void execute_noMatch_noPatientFound() {
 
     @Test
     public void toStringMethod() {
-        NameContainsKeywordsPredicate predicate = new NameContainsKeywordsPredicate(Arrays.asList("keyword"));
-        FindCommand findCommand = new FindCommand(predicate);
-        String expected = FindCommand.class.getCanonicalName() + "{predicate=" + predicate + "}";
+        PatientNameContainsKeywordsPredicate predicate =
```

Comment:
I like how you renamed the variables to follow best variable naming practices. Nice work!